<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="text_view"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionAlign_Justify"/>
   <addaction name="actionAlign_Center"/>
   <addaction name="actionAlign_Left"/>
   <addaction name="actionAlign_Right"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/text_edit_icons/save.png</normaloff>
     <normalon>:/new/prefix1/text_edit_icons/save.png</normalon>:/rec/text_edit_icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current file</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/text_edit_icons/file (1).png</normaloff>:/rec/text_edit_icons/file (1).png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open new file</string>
   </property>
  </action>
  <action name="actionAlign_Justify">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/text_edit_icons/justify-align.png</normaloff>
     <normalon>:/new/prefix1/text_edit_icons/justify-align.png</normalon>:/rec/text_edit_icons/justify-align.png</iconset>
   </property>
   <property name="text">
    <string>Align Justify</string>
   </property>
  </action>
  <action name="actionAlign_Center">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/text_edit_icons/center-alignment.png</normaloff>:/rec/text_edit_icons/center-alignment.png</iconset>
   </property>
   <property name="text">
    <string>Align Center</string>
   </property>
  </action>
  <action name="actionAlign_Left">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/text_edit_icons/left-alignment.png</normaloff>
     <normalon>:/new/prefix1/text_edit_icons/left-alignment.png</normalon>:/rec/text_edit_icons/left-alignment.png</iconset>
   </property>
   <property name="text">
    <string>Align_Left</string>
   </property>
  </action>
  <action name="actionAlign_Right">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/rec/text_edit_icons/right-alignment.png</normaloff>:/rec/text_edit_icons/right-alignment.png</iconset>
   </property>
   <property name="text">
    <string>Align Right</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
